### PR TITLE
feat: add pnpm function in binaries

### LINKED ISSUE #216: feat: add pnpm option in Binaries
Hi Trevor @tabrindle,

At webpack-cli, we need information of `pnpm` installations of user submitting issues with `pnpm` as package manager.
Raised this issue to know whether this would be a welcome addition. If so, I can create a PR for the same 😄.

PR at webpack-cli for reference: 
https://github.com/webpack/webpack-cli/pull/3072
https://github.com/webpack/webpack-cli/pull/3072/files

Thanks and Regards,
Rishabh Chawla.

### PR DESCRIPTION
**Pull request by rishabh3112**
Fixes #216

### CODE REVIEW

**Comment by gengjiawen** [comment]
ping @tabrindle

File: src/presets.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 module.exports = {
   defaults: {
     System: ['OS', 'CPU', 'Memory', 'Container', 'Shell'],
-    Binaries: ['Node', 'Yarn', 'npm', 'Watchman'],
+    Binaries: ['Node', 'Yarn', 'npm', 'Watchman', 'pnpm'],
```
Comment (gengjiawen) [inline src/presets.js:4]: Nit: maybe before Watchman

**Comment by tjx666** [comment]
@rishabh3112 need this!!!

**Comment by gengjiawen** [comment]
@tabrindle Friendly ping.

**Review by tabrindle — APPROVED** [review]

**Comment by alexander-akait** [comment]
@tabrindle Hello and friendly ping, can you look this, pnpm is very popular and we need to get it from users, thank you

**Comment by gengjiawen** [comment]
@rishabh3112 Can you do a rebase ?

**Comment by rishabh3112** [comment]
Yes, will do.

**Comment by rishabh3112** [comment]
@gengjiawen please check.

**Comment by gengjiawen** [comment]
Thx for the contribution.

**Comment by nanianlisao** [comment]
@gengjiawen @rishabh3112 
Can you change @types/envinfo by the way

**Pull request merged**